pyteal/ast/subroutine.py

```diff
@@ -325,14 +385,17 @@ def __teal__(self, options: "CompileOptions"):
 
         3. (ABI, or a special case in by-value) In this case, the storage of an ABI value are loaded
             to the stack and will be stored in a local ABI value for subroutine evaluation
+
+        4. (ABI output keyword argument, or by-ref ABI value) In this case of returning ABI values, we do not place
```

Comment:
nit: ABI output arguments aren't really "by-ref", since they're ouput only

Reply:
yes that is also a segment from the earlier implementation 🤦 should have modify that too

Good catch